### PR TITLE
Adding dropdown selector for supported hash algorithms

### PR DESCRIPTION
**Pull request by uidoyen**

### CODE REVIEW

File: ui/packages/ui/src/app/components/formHelpers/FormMaskHashSaltComponent.tsx [hunk 1]
```diff
@@ -86,21 +90,52 @@ export const FormMaskHashSaltComponent: React.FunctionComponent<IFormMaskHashSal
...
+  };  
+
+  const onSelect = (e, selection: boolean, isPlaceholder: boolean) => {
+    if (isPlaceholder) {
+      clearSelection();
+    }
+    else {
+      setSelected(selection)
+      setOpen(false)
+      let newVal = getColsValue(field.value)+"&&"+selection+"||"+getSaltValue(field.value);
```
Comment (mdrillin) [inline ui/packages/ui/src/app/components/formHelpers/FormMaskHashSaltComponent.tsx:109]: Local build is giving an error - need to change to 'const newVal = ...'

**Review by mdrillin — CHANGES_REQUESTED** [review]
Couple minor things.  I'm getting a build error on line 109 of FormMaskHashSaltComponent.  Also, looks like the layout of the component needs some adjusting - the 'Salt' label is above the input at my default full screen : 

specialProps

File: ui/packages/ui/src/app/components/formHelpers/FormMaskHashSaltComponent.tsx (hunk 1 above)
Comment (uidoyen) [reply ui/packages/ui/src/app/components/formHelpers/FormMaskHashSaltComponent.tsx]: fixed.

**Review by indraraj — APPROVED** [review]
Tested Locally, Looks good.

**Pull request merged**